File: PCMania/src/Computador/SistemaOperacional.java
package Computador;

public class SistemaOperacional {
    private String nome;
    private int tipo;


    //Setters
    public void setTipo(int tipo) {
        this.tipo = tipo;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    //Getters
    public String getNome() {
        return nome;
    }

    public int getTipo() {
        return tipo;
    }
}

File: POO_CPP/PCManiaCPP/src/Computador/SistemaOperacional.cpp
#include "SistemaOperacional.h"

SistemaOperacional::SistemaOperacional(){}

void SistemaOperacional::setTipo(int tipo){
    this->tipo = tipo;
}

void SistemaOperacional::setNome(string nome){
    this->nome = nome;
}

int SistemaOperacional::getTipo(){
    return tipo;
}

string SistemaOperacional::getNome(){
    return nome;
}
